tests/test_camera.py

```diff
@@ -99,14 +99,16 @@ def test_write(self):
         input_video.release()
```

Comment:
Why did `test_video.mp4` change in this update? Is it maybe rewritten anywhere in the unit tests?

Reply:
The writer output a video using a different encoder, resulting in some minor variation,  so I replaced the input video with the one created to match the encoding style

Reply:
Makes sense. So now there should be no more changes after running the tests, right?